pyccel/codegen/printing/pycode.py

```diff
@@ -32,6 +35,12 @@ def _construct_header(func_name, args):
     return pattern.format(name=func_name, args=args)
 
 #==============================================================================
+
+import_target_swap = {
+        'numpy' : {'double' : 'float64', 'product' : 'prod'},
+        'numpy.random' : {'random' : 'rand'}
+        }
```

Comment:
Could you explain how you use this dictionary? A line of comment may be useful here.

Reply:
It maps objects to equivalent objects. For example:
```
In [2]: from numpy import double

In [3]: a = double(4)

In [4]: print(type(a))
<class 'numpy.float64'>
```
`numpy.double` is just an alias for `numpy.float64`, internally we will use `NumpyFloat64`, so we would print:
```
from numpy import double
a = float64(4)
```
The import is collected from the user's code, but the function call is printed from our class. The dictionary therefore maps imported objects to the alias which is used internally in pyccel.
I will try to add a clear comment.

Reply:
Fixed in b33d098

Reply:
Thanks!